src/Message/Envelope.php

```diff
@@ -23,14 +20,6 @@ public function getMessage(): MessageInterface
         return $this->message;
     }
 
-    public function withMessage(MessageInterface $message): self
```

Comment:
why?

Reply:
What's the real-world use case? I don't see any, that's why I removed this method.